### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by Steven4294** [wowthemesnet/mediumish-theme-jekyll#154]
is there a dark mode version of the mediumish.scss?

**Comment by PeterHdd** [comment]
No, but you can use this tutorial add it to your website, and customize it:

https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8

**Comment by thatvirtualboy** [comment]
This would be great
